packages/gatsby-theme/src/blocks/ContactBlock/index.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from "prop-types"
+import Contact from "../../components/Contact"
```

Comment:
Please add the Contact component to the [component index.js file](https://github.com/octahedroid/drupal-gatsby-theme/blob/master/packages/gatsby-theme/src/components/index.js) and update this import (see Button component example)
```suggestion
import { Contact } from "ui"
```